### PR TITLE
Util change: createElementNS removed/createElement set in it's place

### PR DESCRIPTION
**Pull request by nickolasnikolic**
Related issue: #XXXX

**Description**

Probably an accident, but an error was put into dev branch element creation. I fixed it.

Untested (I'm not able to get tests running) I beg that you test my code in case of other createElementNS instances, I noticed a few in the main branch but don't know the system well enough to troubleshoot. This util I can fix, though.

### CODE REVIEW

**Comment by nickolasnikolic** [comment]
Thank you for your work! Happy to help!

File: src/utils.js [hunk 1]
```diff
@@ -68,9 +68,13 @@ function createElementNS( name ) {
 
 }
 
+function createElement( name ){
+	return document.createElement( name );
+}
+
 function createCanvasElement() {
 
-	const canvas = createElementNS( 'canvas' );
```
Comment (Mugen87) [inline src/utils.js:73]: > but an error was put into dev branch element creation.

Why is using `createElementNS()` here an error. It was implemented like this in purpose, see #9041.

**Comment by nickolasnikolic** [comment]
I understand the confusion, but I thought it only used in svg rendering. I saw that it was a relatively new commit, I thought it probably a typo with autocomplete. [Document: createElement() method - Web APIs | MDN (mozilla.org)](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement)

File: src/utils.js (hunk 1 above)
Comment (nickolasnikolic) [reply src/utils.js:73]: I understand the confusion, but I thought it only used in svg rendering. I saw that it was a relatively new commit, I thought it probably a typo with autocomplete. [Document: createElement() method - Web APIs | MDN (mozilla.org)](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement) Incidentally, I am not sure the other pull request was correct. If I am in error, I apologize; trying to help.

Comment (mrdoob) [reply src/utils.js:73]: I don't think it's a typo...

**Pull request closed without merging**